maniphttp/manipulator.go

```diff
@@ -399,9 +399,9 @@ func (s *httpManipulator) Count(mctx manipulate.Context, identity elemental.Iden
 	return mctx.Count(), nil
 }
 
-func (s *httpManipulator) makeAuthorizationHeaders() string {
+func (s *httpManipulator) makeAuthorizationHeaders(username, password string) string {
 
-	return s.username + " " + s.currentPassword()
+	return username + " " + password
```

Comment:
Wondering why we are not using a standard here (e.g. http basic auth)?

This prevents you from having a space in your password as we are using that as a delimiter.

Reply:
This was basic done for Authorization: Bearer Token 
(that's the standard for most OAUTH related things). The fact that manipulate could potentially support username/password is an option.

In most our use cases it is Bearer tokens.

Reply:
👍